ironfish/src/merkletree/merkletree.test.ts

```diff
@@ -146,6 +208,59 @@ describe('Merkle tree', function () {
         '<<<<a|b-0>|<c|d-0>-1>|<<e|f-0>|<g|h-0>-1>-2>|<<<i|i-0>|<i|i-0>-1>|<<i|i-0>|<i|i-0>-1>-2>-3>',
       ],
     ])
+
+    await tree.addBatch('jklmnopqrstuvwxyz')
+    await expect(tree).toHaveLeaves(
+      'abcdefghijklmnopqrstuvwxyz',
+      [
+        1, 1, 2, 2, 4, 4, 7, 7, 8, 8, 12, 12, 13, 13, 15, 15, 16, 16, 21, 21, 22, 22, 24, 24,
+        25, 25,
+      ],
+    )
+    await expect(tree).toHaveNodes([
+      [1, Side.Left, 3, '<c|d-0>'],
```

Comment:
I think this format is pretty difficult to read. You think there is a way we could assert something that makes this easier to reason about? Maybe some helper functions to construct the hashes?